server/config/config.go

```diff
@@ -780,6 +786,8 @@ const (
 	defaultStoreLimitMode              = "manual"
 	defaultEnableJointConsensus        = true
 	defaultEnableCrossTableMerge       = true
+	defaultHotRegionsWriteInterval     = 20 * time.Minute
```

Comment:
It should be the same as `conf/config.toml`.  10 time.Minute and defaultHotRegionsResevervedDays = 7